pulp_ansible/app/migrations/0034_deprecation_content.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 2.2.24 on 2021-07-07 17:46
+
+from django.db import migrations, models, transaction
+import django.db.models.deletion
+
+
+def migrate_data_from_old_model_to_new_model_up(apps, schema_editor):
+    """ Move objects from AnsibleCollectionDeprecated to NewAnsibleCollectionDeprecated."""
+    AnsibleCollectionDeprecated = apps.get_model('ansible', 'AnsibleCollectionDeprecated')
+    NewAnsibleCollectionDeprecated = apps.get_model('ansible', 'NewAnsibleCollectionDeprecated')
+    RepositoryContent = apps.get_model('core', 'RepositoryContent')
+    repo_content = []
+    with transaction.atomic():
+        for deprecated_collection in AnsibleCollectionDeprecated.objects.all():
+            NewAnsibleCollectionDeprecated(
+                namespace=deprecated_collection.collection.namespace,
+                name=deprecated_collection.collection.name,
+                pulp_type="ansible.collection_deprecation",
+                pulp_id=deprecated_collection.pulp_id,
+            ).save()
+            repo_content.append(RepositoryContent(
+                repository=deprecated_collection.repository_version.repository,
+                content_id=deprecated_collection.pk,
+                version_added=deprecated_collection.repository_version,
+            ))
+
+        RepositoryContent.objects.bulk_create(repo_content)
+        AnsibleCollectionDeprecated.objects.all().delete()
```

Comment:
@bmbouter @daviddavis @gerrod3 the PR is almost ready, the only thing I couldn't make work is the data migration.
Have you seen something like this before?
```
django.db.utils.OperationalError: cannot ALTER TABLE "core_repositorycontent" because it has pending trigger events
```

Reply:
I believe I need to divide into 2 migrations:
https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/docs/ref/migration-operations.txt#L390-L395